common.tf

```diff
@@ -94,7 +94,7 @@ module "dr2_kms_key" {
       module.ingest_folder_opex_creator_lambda.lambda_role_arn,
       module.ingest_upsert_archives_folder_lambda.lambda_role_arn
     ], local.additional_user_roles)
-    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/IntgTerraformRole"]
+    ci_roles      = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${title(local.environment)}TerraformRole"]
```

Comment:
Couldn't you use `local.environment_title` here instead?

Reply:
Good spot, I forgot about that. I've changed it. Same for the one below.